src/display/macros.rs

```diff
@@ -101,25 +36,41 @@ macro_rules! group {
 
         builder.build()
     }};
+}
 
-    // Divide items into rows
-    ( $($table:expr), *; $N:expr) => {{
-        let tables = &[$($table.to_string(),)*];
-        let groups = tables
-            .chunks($N)
-            .map(|chunk| {
-                chunk.iter()
-                    .map(ToString::to_string)
-                    .chain(std::iter::repeat(String::default()))
-                    .take($N)
-            });
+/// Creates a parent [`Table`] with [`std::fmt::Display`] arguments nested within.
+///
+/// col! allows several tables to be displayed vertically.
+///
+/// Companion to [`row!`].
+///
+/// # Examples
+/// ```rust,no_run
+/// # use tabled::{row, col, Table};
+/// # let (table1, table2, table3) = (Table::new(&[String::new()]), Table::new(&[String::new()]), Table::new(&[String::new()]));
+/// let new_table = col![table1, table2];
+/// let new_table_of_clones = col![table1; 3];
+/// let columns_and_rows = col![
+///     table1,
+///     row![table2, table3]
+/// ];
+/// ```
+#[macro_export]
+macro_rules! col {
+    // Vertical
+    ( $($table:expr), * $(,)? ) => {{
+        let mut builder = Table::builder([ $($table.to_string(),)* ]);
+        builder.remove_columns();
```

Comment:
(can be ignored)

I think creating `Builder::default` and then calling `add_record` be more efficient.
But that's OK.